### ISSUE TITLE
A question about image size, label size, stride setting

### ISSUE THREAD

**Issue by chun-yuan** [tegg89/SRCNN-Tensorflow#50]
Dear Author:
         I want to ask, how do you set your parameters (image size=33, label size=21, stride=14)?
         I have a new training dataset want to use your code to execute. There are 64 images in my new training dataset. All of them  original pixel are 282×375. All of test image pixels are 282*375. There are six images in my test dataset. 
          Now, I have problem in image size, label size, stride setting. Can I have a discussion with you.?
         My email is arthur90112@gmail.com.
Sincerely,
Chun-Yuan Shin
